spark/src/main/scala/io/substrait/spark/logical/ToSubstraitRel.scala

```diff
@@ -64,6 +66,10 @@ class ToSubstraitRel extends AbstractLogicalPlanVisitor with Logging {
 
   private val existenceJoins = scala.collection.mutable.Map[Long, SExpression.InPredicate]()
 
+  private var _rddLimit = 100
+  def rddLimit: Int = _rddLimit
```

Comment:
it might be good idea to check whether the RDD has more than the `rddLimit` rows and log a warning message in order to notify users of the fact and inform them that there is a modifiable limit